Documentation/Includes.rst.txt

```diff
@@ -38,5 +38,5 @@
    maintained by the community anymore. Looking for a stable version?
    Use the version switch on the top left.
 
-   You can order Extended Long Term Support (ELTS) here:
-   `TYPO3 ELTS <https://typo3.com/services/extended-support-elts>`__.
+   There is no further ELTS support. It is strongly recommended updating your
```

Comment:
would suggest 

```
- It is strongly recommended updating your
+ It is strongly recommended to update your
```

Ah, ok, I see that is already used this way https://docs.typo3.org/m/typo3/reference-typoscript/6.2/en-us/

Reply:
Yup, just picking it up from the last one.